src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -51,6 +52,18 @@ class PeerActor[R <: HandshakeResult](
 
   val peerId: PeerId = PeerId.fromRef(self)
 
+  // The actor logs incoming messages, which can be quite verbose even for DEBUG mode.
+  // ActorLogging doesn't support TRACE, but we can push more details if trace is enabled.
+  val isTraceEnabled = {
```

Comment:
I thought about something like to workaround Akka limitations
```scala
class PeerActorLogger extends io.iohk.ethereum.utils.Logger {}
class PeerActor {
	val logger = new PeerActorLogger

	//...

	logger.log.debug(foo)
	logger.log.trace(bar)
}

```

Reply:
Can do that too, I just didn't want to mix it that half of the messages are logged with the `ActorLogging` machinery which has a different configuration than the `Logger` stuff, except for this one. Best would be to get rid of `ActorLogging` altogether.

Reply:
Pushed a version of that too, but just for messages.